roles/kubernetes-apps/metrics_server/templates/resource-reader.yaml.j2

```diff
@@ -17,12 +17,3 @@ rules:
       - get
       - list
       - watch
-  - apiGroups:
-      - "apps"
-    resources:
-      - deployments
-    verbs:
-      - get
-      - list
-      - update
-      - watch
```

Comment:
lgtm according to https://github.com/kubernetes-sigs/metrics-server/blob/93b9ddc8cefa40d0fc7fc13ba741220a6a463bc0/manifests/base/rbac.yaml#L48-L64